contracts/ColonyAuthority.sol

```diff
@@ -80,6 +78,8 @@ contract ColonyAuthority is CommonAuthority {
 
     // Added in colony v3
     addRoleCapability(ROOT_ROLE, "updateColonyOrbitDB(string)");
+    addRoleCapability(ROOT_ROLE, "setArbitrationRole(uint256,uint256,address,uint256,bool)");
+    addRoleCapability(ARCHITECTURE_SUBDOMAIN_ROLE, "setArbitrationRole(uint256,uint256,address,uint256,bool)");
```

Comment:
I don't see the benefit of grouping this functionality in chronological order -- it seems better for readability if these stay with the original grouping.

Reply:
The benefit is understanding what domain role capabilities are there for each colony version, which arguably is rather important when for example troubleshooting a specific colony version issues related to permissions.

Reply:
Mm ok, I can buy that.